contrib/cmd/memfd-bind/README.md

```diff
@@ -0,0 +1,67 @@
+## memfd-bind ##
```

Comment:
Perhaps we need the main README to refer to this doc.

Reply:
I've added a reference in the section mentioning `runc_nodmz` and `RUNC_DMZ=legacy`.